### PR TITLE
add content to working with github

### PR DESCRIPTION
**Pull request by lourieder**

### CODE REVIEW

File: doc/source/git_account.rst [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
+Commiting to a change
+=====================
+
+A commit records changes on your repository.
+If you make a commit you have to attach a brief message describing the changes.  
+
+Some best practices for commits: 
+
+    - make small single-pupose commits 
+    - write short datailed commit meggsages
```
Comment (gtema) [inline doc/source/git_account.rst:35]: many typos in the word "message"

**Review by gtema — CHANGES_REQUESTED** [review]
Maybe reuse more text as inspiration for sentence phrasing from github docs?

There are actually plenty of articles in internet which describes this topics in details:
https://www.gitkraken.com/learn/git/best-practices/git-commit-message
https://medium.com/swlh/writing-better-commit-messages-9b0b6ff60c67
https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/
https://www.toolsqa.com/git/git-fork/
...

We do not need to write the whole article about git, but it should content more text

**Comment by kucerakk** [comment]
content merged with https://github.com/opentelekomcloud-docs/docs_on_docs/pull/20

**Comment by kucerakk** [comment]
content merged with https://github.com/opentelekomcloud-docs/docs_on_docs/pull/20

**Pull request closed without merging**